seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/split/FileSourceSplitEnumerator.java

```diff
@@ -37,12 +37,12 @@ public class FileSourceSplitEnumerator implements SourceSplitEnumerator<FileSour
     private Set<FileSourceSplit> assignedSplit;
```

Comment:
Please keep this file change out of your submission, give you a advice, you can revert this file and keep this file the same as dev branch.